src/owner/services/owner.service.ts

```diff
@@ -0,0 +1,38 @@
+import { CommandBus } from '@nestjs/cqrs';
+import { Injectable } from '@nestjs/common';
+import { OwnerIdDto } from '../dtos/owner.dto';
+import { UpdateOwnerDto } from '../dtos/update-owner.dto';
+import { RegisterOwnerDto } from '../dtos/register-owner.dto';
+import { RegisterCommand } from '../commands/impl/register-owner.command';
+import { RemoveOwnerCommand } from '../commands/impl/remove-owner.command';
+import { UpdateOwnerCommand } from '../commands/impl/update-owner.command';
+
+
+@Injectable()
+export class OwnerService {
```

Comment:
Is een service tussen de `Controller` en de `CommandBus` nog nodig? Wat voegt deze 'laag' toe?